frontend/src/components/views/RangeIndicatorView.jsx

```diff
@@ -0,0 +1,37 @@
+import { pythagorean, round } from '../../utils/math_utils';
+import { convertToRenderCoordinates, tileConfig } from '../../utils/render_utils';
+
+export const RangeIndicatorView = ({ tower, gameMap }) => {
+    const { minRange, maxRange } = tower;
+    const SQUARE_SIZE = tileConfig.width/3;
```

Comment:
Ah, sorry, I was wrong here. You call `convertToRenderCoordinates()` later, so this was correct before.
```suggestion
    const SQUARE_SIZE = 1/3;
```